### PR TITLE
Update project-setup to correct JDK version.

### PR DESCRIPTION
**Pull request by Axodouble**
Updated the JDK, as 17 is no longer supported.

### CODE REVIEW

**Comment by Strokkur424** [comment]
I am about to do a PR which also updates the picture above

**Comment by lynxplay** [comment]
Just include the picture change in this PR 👍 LGTM otherwise

**Comment by Axodouble** [comment]
> I am about to do a PR which also updates the picture above

Would it not be handier to add it onto this PR?

**Comment by Strokkur424** [comment]
Can I even edit this pr here?

**Comment by lynxplay** [comment]
Just post the picture here, they can include it 👍

**Comment by Strokkur424** [comment]
Very well, here you go:
new-project-paper

**Comment by lynxplay** [comment]
Merci <3

**Comment by Strokkur424** [comment]
Looks Good 👍

File: docs/paper/dev/getting-started/project-setup.mdx [hunk 1]
```diff
@@ -185,7 +185,7 @@ You will be asked to provide some information about your project.
...
 | **Artifact ID**       | The artifact ID of your project. This is used for Maven and Gradle. This is usually the name of your project. This is usually the same as the `Name` field.                                                                                                          |
 | **Version**           | The version of your project. This is used for Maven and Gradle. This is usually `1.0-SNAPSHOT` and does not really matter for now.                                                                                                                                   |
-| **JDK**               | The JDK you want to use. This can be anything from Java 17 and above.                                                                                                                                                                                                |
+| **JDK**               | The JDK you want to use. This can be anything from Java 21 and above.                                                                                                                                                                                                |
```
Comment (zlataovce) [inline docs/paper/dev/getting-started/project-setup.mdx:188]: this should probably be changed to refer to the `PAPER_JAVA_MIN` property: https://github.com/PaperMC/docs/blob/main/CONTRIBUTING.md#magic-value-handling

Comment (Axodouble) [reply docs/paper/dev/getting-started/project-setup.mdx]: Good idea, just committed an attempt for it.

Comment (zlataovce) [reply docs/paper/dev/getting-started/project-setup.mdx]: looks good, just trim the excess space from the table field, so the end lines up

**Comment by Strokkur424** [comment]
Looks Good² 👍

**Comment by MiniDigger** [comment]
why does the new screenshot sets the paper-plugin.yml checkbox?
we do not recommend people to use that unless the need it, especially beginners that would use this guide

**Comment by Strokkur424** [comment]
That is a fair point. That was my bad, as I accidentally had it ticked and forgot about it

**Pull request merged**